Tutorials/Extension_topics/Tuto_A.1_Parameter_estimation_for_compact_object_mergers_with_LALInference.ipynb

```diff
@@ -264,17 +269,9 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Parameter estimation file found in: /home/jerry/GW/odw-2024/Tutorials/Extension_topics/tuto_A.1/GW190828_063405/GW190828_063405.h5\n",
+      "Parameter estimation file found in: /Users/mdubois/Projects/LVK/GWOSC/ODW/odw/Tutorials/Extension_topics/tuto_A.1/GW190828_063405/GW190828_063405.h5\n",
```

Comment:
Same with this exposed path

Reply:
there are a lot more after this one too

Reply:
TBH I just wanted to point it out. I think this issue is out of scope for this PR and can be dealt with in a different PR. I'm resolving this thread.